interfaces/acados_c/ocp_nlp_interface.c

```diff
@@ -533,6 +533,10 @@ int ocp_nlp_dims_get_from_attr(ocp_nlp_config *config, ocp_nlp_dims *dims, ocp_n
     {
         return dims->nx[stage];
     }
+    else if (!strcmp(field, "pi"))
+    {
+        return dims->nx[stage];
```

Comment:
the size of `pi[stage]` is `nx[stage+1]`

Reply:
good catch, thanks!